TestFramework.h

```diff
@@ -398,31 +397,22 @@ namespace TFW
 
 	inline QString comareFilesGZ(QString actual, QString expected)
 	{
-		//init buffer
-		QByteArray buffer(1024, ' ');
-
 		//make file names absolute
 		actual = QFileInfo(actual).absoluteFilePath();
 		expected = QFileInfo(expected).absoluteFilePath();
 
 		//open streams
-		gzFile streama = gzopen(actual.toUtf8().data(),"rb");
-		if (streama == NULL)  return "Could not open file '" + actual + "' for reading!";
-		gzFile streame = gzopen(expected.toUtf8().data(),"rb");
-		if (streame == NULL)  return "Could not open file '" + expected + "' for reading!";
+		VersatileFile streama(actual);
+		streama.open();
+		VersatileFile streame(actual);
+		streame.open();
 
```

Comment:
VersatileFile will uncompress only remote files, local files are handled by VcfFile class. I did not know we have local GZ files somewhere else. I can easily add the support for local GZ files as well

Reply:
not anymore. See my other pull requests :)

Reply:
oh, I see :)